src/syntax/lexer.ml

```diff
@@ -544,9 +509,8 @@ and code_string lexbuf open_braces =
 	| "'" ->
 		add "'";
 		let pmin = lexeme_start lexbuf in
-		let pmax = (try string2 lexbuf with Exit -> error Unterminated_string pmin) in
+		ignore(try string2 lexbuf with Exit -> error Unterminated_string pmin);
```

Comment:
Do we really write these without a space after `ignore`? x)

Reply:
Welp that's how it's written in all other places in this file, even though I would normally put a space before arguments... But that got me thinking it should be `(try ignore(string2 lexbuf) with Exit -> error Unterminated_string pmin);` just like the line above ^^